### PR TITLE
feat(registry-artifacts): import export commands

### PR DESCRIPTION
**Pull request by wtrocki**
Verification:
```
rhoas service-registry artifact list > tmp.list
rhoas service-registry artifact export --output-file=export.zip

rhoas service-registry create --name=new-export
rhoas service-registry artifact import --file=export.zip
rhoas service-registry artifact list > tmp.list2

diff tmp.list tmp.list2 | wc

```

### CODE REVIEW

**Comment by wtrocki** [comment]
Related PR(bug): https://github.com/redhat-developer/app-services-sdk-go/pull/272

**Comment by wtrocki** [comment]
Ready for review

**Review by rkpattnaik780 — COMMENTED** [review]
Import command doesn't seem to be working, imported a zip with 2 artifacts ( group = default ). Can't see the imported artifacts.

Screenshot from 2021-09-07 16-11-55

File: pkg/cmd/registry/artifact/migrate/import.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+				return errors.New(opts.localizer.MustLocalize("artifact.cmd.common.error.noServiceRegistrySelected"))
+			}
+
+			opts.registryID = cfg.Services.ServiceRegistry.InstanceID
+			return runImport(opts)
+		},
+	}
+	cmd.Flags().StringVar(&opts.file, "file", "", opts.localizer.MustLocalize("artifact.common.file.location"))
+	cmd.Flags().StringVar(&opts.registryID, "instance-id", "", opts.localizer.MustLocalize("artifact.common.instance.id"))
+	_ = cmd.MarkFlagRequired("file")
```
Comment (rkpattnaik780) [inline pkg/cmd/registry/artifact/migrate/import.go:72]: Marking `file` as required doesnt allow us to pass file path as argument

File: pkg/cmd/registry/artifact/migrate/export.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+			if !cfg.HasServiceRegistry() {
+				return errors.New(opts.localizer.MustLocalize("artifact.cmd.common.error.noServiceRegistrySelected"))
+			}
+
+			opts.registryID = cfg.Services.ServiceRegistry.InstanceID
+			return runExport(opts)
+		},
+	}
+	cmd.Flags().StringVar(&opts.file, "file", "", opts.localizer.MustLocalize("artifact.common.file.location"))
```
Comment (rkpattnaik780) [inline pkg/cmd/registry/artifact/migrate/export.go:67]: Should it be `--output-file`?
Context: #1008

**Review by rkpattnaik780 — CHANGES_REQUESTED** [review]

File: pkg/cmd/registry/artifact/migrate/export.go (hunk 2 above)
Comment (wtrocki) [reply pkg/cmd/registry/artifact/migrate/export.go:67]: @craicoverflow what would be your take on this?

File: pkg/cmd/registry/artifact/migrate/import.go (hunk 1 above)
Comment (wtrocki) [reply pkg/cmd/registry/artifact/migrate/import.go:72]: Bad copy paste :)

File: pkg/cmd/registry/artifact/migrate/export.go (hunk 2 above)
Comment (craicoverflow) [reply pkg/cmd/registry/artifact/migrate/export.go:67]: If the contents from the `export` command being saved in the file, it should be `--output-file`.

I notice this is not using ['artifact.common.message.file.location'](https://github.com/redhat-developer/app-services-cli/blob/1165a3fde7562851355f55a1940e1e059a4f4ea4/pkg/localize/locales/en/cmd/artifact.en.toml#L103) like other output file flags are using.

Comment (craicoverflow) [reply pkg/cmd/registry/artifact/migrate/export.go:67]: In saying that it is obvious from the context this time (`service-account create` is less clear). But we should be consistent.

**Review by rkpattnaik780 — APPROVED** [review]

**Comment by wtrocki** [comment]
Import were identified as backend problem. Merging so others can try to fix that on their side.

**Pull request merged**